addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -698,26 +698,24 @@ function factory(dependencies) {
             } else {
                 const authorName = author.nameOrDisplayName;
                 if (channel.channel_type === 'channel') {
-                    // hack: notification template does not support OWL components,
-                    // so we simply use their template to make HTML as if it comes
-                    // from component
-                    const channelIcon = this.env.qweb.renderToString('mail.ThreadIcon', {
-                        env: this.env,
-                        thread: channel,
-                    });
+                    const channelIcon = channel.public === 'private' ? '<i class="fa fa-lock"/> '
+                                                                     : '<i class="fa fa-hashtag"/> ';
                     const channelName = owl.utils.escape(channel.displayName);
                     const channelNameWithIcon = channelIcon + channelName;
                     notificationTitle = _.str.sprintf(
-                        this.env._t("%s from %s"),
+                        this.env._t("<b>%s from %s</b>"),
```

Comment:
The `<b>` formatting is the same in both cases, I think it should be moved once to `notificationContent`.